docs/source/developer/signals.rst

```diff
@@ -0,0 +1,14 @@
+=======
+Signals
+=======
+
+``radius_accounting_success``
+-----------------------------
+
+**Path**: ``openwisp_radius.signals.radius_accounting_success``
+
+**Arguments**:
+
+- ``accounting_data``: accounting information
+
+This signal is emitted every time accounting information is added or updated.
```

Comment:
Let's make sure the following concepts are extra clear:

- the signal is emitted every time the accounting REST API endpoint completes successfully
- the signal is emitted right before the response is returned
- accounting_data is a `dict`